### PR TITLE
Use requests locally instead of wget remotely

### PR DESCRIPTION
**Pull request by zmc**
... in _update_deb_package_list_and_install()

Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

**Comment by zmc** [comment]
http://pulpito.ceph.com/zack-2015-05-28_09:26:50-teuthology:ceph-master---basic-multi/

**Comment by ktdreyer** [comment]
this is the :bomb:

**Comment by zmc** [comment]
test this please

**Comment by dmick** [comment]
The pulpito jobs appear to still be using wget?...maybe I'm misunderstanding what those are supposed to represent

**Comment by zmc** [comment]
@dmick ah, the first job is using RHEL; I didn't update that codepath. I will.

The second job on Ubuntu isn't using wget for this functionality.

File: teuthology/task/install.py [hunk 1]
```diff
@@ -336,7 +335,8 @@ def _update_deb_package_list_and_install(ctx, remote, debs, config):
     remote.run(args=['sudo', 'apt-get', 'update'], check_status=False)
     remote.run(
         args=[
-            'sudo', 'DEBIAN_FRONTEND=noninteractive', 'apt-get', '-y', '--force-yes',
+            'sudo', 'DEBIAN_FRONTEND=noninteractive', 'apt-get', '-y',
+            '--force-yes',
```
Comment (ktdreyer) [inline teuthology/task/install.py]: Looks like a separate change, here?

**Comment by ktdreyer** [comment]
ticket http://tracker.ceph.com/issues/11630 is also related to this.

File: teuthology/task/install.py (hunk 1 above)
Comment (zmc) [inline teuthology/task/install.py]: Just a line-wrapping change to make the linter happier.

**Comment by zmc** [comment]
http://pulpito.ceph.com/zack-2015-05-29_09:48:10-teuthology:ceph-master---basic-multi/

File: teuthology/task/install.py (hunk 1 above)
Comment (ktdreyer) [inline teuthology/task/install.py]: You know, if it was Nit-Picking Friday I would say it needs to be a separate commit :D

Comment (zmc) [inline teuthology/task/install.py]: I guess I can just remove that change if you feel strongly about it?

**Comment by zmc** [comment]
test this please

File: teuthology/task/install.py (hunk 1 above)
Comment (dmick) [inline teuthology/task/install.py]: overruled!  there's too damn many PRs already

**Comment by andrewschoen** [comment]
This seems sane to me, are we waiting on anything else before merge on this?

**Comment by zmc** [comment]
I'd been wondering the same thing.
